bugsnag-android-core/src/test/java/com/bugsnag/android/LaunchCrashDeliveryTest.kt

```diff
@@ -0,0 +1,136 @@
+package com.bugsnag.android
+
+import com.bugsnag.android.BugsnagTestUtils.generateConfiguration
+import com.bugsnag.android.BugsnagTestUtils.generateEvent
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import java.io.File
+import java.lang.Thread
+import java.nio.file.Files
+import java.util.concurrent.atomic.AtomicInteger
+
+/**
+ * Verifies that crashes on launch are sent synchronously and other crashes are not.
+ */
+class LaunchCrashDeliveryTest {
+
+    private lateinit var storageDir: File
+    private lateinit var errorDir: File
+
+    @Before
+    fun setUp() {
+        storageDir = Files.createTempDirectory("tmp").toFile()
+        storageDir.deleteRecursively()
```

Comment:
Since [`Files.createTempDirectory()`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory(java.lang.String,%20java.nio.file.attribute.FileAttribute...)) creates a new directory with a different suffix every run, it doesn't look like this line does anything?

Reply:
Yep, this doesn't seem to be necessary - I've removed this line.